test/test_request.py

```diff
@@ -0,0 +1,24 @@
+import pytest
```

Comment:
The test cases look good but I think there's an existing place for tests of `urllib3.request()`

Reply:
Great, can you show me? I have doubts to the best place to put this test

Reply:
In the file `test/with_dummyserver/test_poolmanager.py` search for `test_top_level_request`. Under that test case should work!

Reply:
We only need the extra `TypeError` case btw, the positive case is already there.

Reply:
Done